neo/Network/RPC/Model/RpcTransaction.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Neo.IO.Json;
+using Neo.Network.P2P.Payloads;
+
+namespace Neo.Network.RPC.Model
+{
+    public class RpcTransaction
+    {
+        public Transaction Transaction { get; set; }
+
+        public UInt256 BlockHash { get; set; }
+
+        public int? Confirmations { get; set; }
+
+        public uint? BlockTime { get; set; }
+
+        public JObject ToJson()
+        {
+            JObject json = Transaction.ToJson();
+            if (Confirmations != null)
+            {
+                json["blockhash"] = BlockHash.ToString();
+                json["confirmations"] = Confirmations;
```

Comment:
What return if is null? should be 0 and i guess that is null (same comment for all nullable values)

Reply:
![image](https://user-images.githubusercontent.com/4393234/61857579-07a63900-aef7-11e9-9290-7777fcdfaa5a.png)
This is consistent with RpcServer response